### DISCUSSION TITLE
Cypress + Storybook 2.0 blog entry - preview falls outside of src/ directory

### DISCUSSION THREAD

**Discussion by yann-combarnous** [cypress-io/cypress#16685] [Component Testing]
Hi,

I attempted to implement the example in blog entry https://www.cypress.io/blog/2021/05/19/cypress-x-storybook-2-0/, and got an error when importing Storybook's preview file in Cypress /support/index.js:

import * as sbPreview from "../../.storybook/preview";

Error: You attempted to import ../../.storybook/preview which falls outside of the project src/ directory

Any tip on how to make this work with Cypress webpack dev server (we use CRA, '@cypress/react/plugins/react-scripts')?

**Comment by hch2904** [comment]
@yann-combarnous Were you able to fix this?
